mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -1,33 +1,37 @@
 CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
 	FIELD field_22734 zOffset I
-	FIELD field_22735 BACKGROUND_LOCATION Lnet/minecraft/class_2960;
-	FIELD field_22736 STATS_ICON_LOCATION Lnet/minecraft/class_2960;
-	FIELD field_22737 GUI_ICONS_LOCATION Lnet/minecraft/class_2960;
-	METHOD method_25290 blit (IIFFIIII)V
+		COMMENT The z offset used by {@link DrawableHelper}.
+	FIELD field_22735 BACKGROUND_TEXTURE Lnet/minecraft/class_2960;
+		COMMENT The texture used by options for background.
+	FIELD field_22736 STATS_ICON_TEXTURE Lnet/minecraft/class_2960;
+		COMMENT The texture of icons used in the stats screen.
+	FIELD field_22737 GUI_ICONS_TEXTURE Lnet/minecraft/class_2960;
+		COMMENT The texture of various icons and widgets used for rendering ingame indicators.
+	METHOD method_25290 drawTexture (IIFFIIII)V
 		ARG 0 x
 		ARG 1 y
 		ARG 2 u
 		ARG 3 v
 		ARG 4 width
 		ARG 5 height
-		ARG 6 texWidth
-		ARG 7 texHeight
-	METHOD method_25291 blit (IIIFFIIII)V
+		ARG 6 textureWidth
+		ARG 7 textureHeight
+	METHOD method_25291 drawTexture (IIIFFIIII)V
 		ARG 0 x
 		ARG 1 y
 		ARG 2 z
 		ARG 3 u
 		ARG 4 v
 		ARG 5 width
 		ARG 6 height
-		ARG 7 texHeight
-		ARG 8 texWidth
+		ARG 7 textureHeight
+		ARG 8 textureWidth
 	METHOD method_25292 drawHorizontalLine (IIII)V
 		ARG 1 x1
 		ARG 2 x2
```

Comment:
```suggestion
		ARG 1 xStart
		ARG 2 xEnd
```

Reply:
No, they are interchangeable, you can swap them around for this one.